packages/datastore-storage-adapter/__tests__/ExpoSQLiteDatabase.test.ts

```diff
@@ -267,7 +284,7 @@ describe('ExpoSQLiteDatabase 2', () => {
 			await expect(instance.batchSave()).resolves.toBeNull();
 		});
 
-		it('multiple save statements. should reject to error when the transaction is unsuccessfull', async () => {
+		it('should resolve to null when successful and provided multiple saveStatements.', async () => {
```

Comment:
Nit: For consistency
```suggestion
		it('should resolve to null when successful and provided multiple save statements.', async () => {
```

Reply:
Got it. Will update that